http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/AppServiceInterpSpec.scala

```diff
@@ -108,34 +152,19 @@ final class AppServiceInterpSpec extends AnyFlatSpec with LeonardoTestSuite with
     }
   }
 
-  it should "fail request if user is not in custom_app_users group" in {
```

Comment:
we should add back the test..

Reply:
*facepalm* I must have ctrl+x not ctrl+c